app/manager-frontend/src/pages/UserHome.jsx

```diff
@@ -4,19 +4,25 @@ import { ActivitySourcesList } from "./ActivitySource/list";
 import { useParams, useNavigate } from "react-router-dom";
 import { useEffect } from "react";
 import { AgentsList } from "./Agent/list";
+import { UsersList } from "./User/list";
 
 export const UserHome = () => {
   const homepageActions = [
     {
       title: "Activity Source",
-      description: "Activity Source description",
+      description: "Activity Source action card description",
```

Comment:
I don't like the text here if it's something the user will see. It's kind of saying what the underlying code is, and the user won't care.

Reply:
I think Jon suggested if we could add some description in the card, thus the placeholder. But for now I have removed it and assigned an icon to the card.